lib/versioning/node/index.ts

```diff
@@ -15,6 +19,13 @@ function getNewValue({
   currentVersion,
   newVersion,
 }: NewValueConfig): string | null {
+  // Try to use codename if the current value is a codename
+  if (findScheduleForCodename(currentValue)) {
+    const newSchedule = findScheduleForVersion(newVersion);
+    if (newSchedule?.codename) {
+      return newSchedule.codename.toLowerCase();
```

Comment:
In this case we would not go through `npm` versioning at all. Could that be a problem?

Reply:
I think it's ok